pkg/prom/agent.go

```diff
@@ -35,7 +35,6 @@ var (
 
 var (
 	DefaultConfig = Config{
-		Global:                 config.DefaultGlobalConfig,
```

Comment:
This was never used so I took it out; when the Global config is unmarshaled by Prometheus it will provide the defaults.